backend/src/hooks/check-has-permission/check-has-permission.hook.ts

```diff
@@ -8,8 +8,10 @@ import {
 import { PermissionsError } from '~/exceptions/exceptions';
 import { checkHasPermission } from '~/helpers/helpers';
 
+type CheckType = 'every' | 'oneOf';
```

Comment:
```suggestion
type CheckPermisssionType = 'every' | 'oneOf';
```
move to the shared/common/types/permission/check-permisssion-type.type.ts